Cargo.toml

```diff
@@ -4,10 +4,15 @@ version = "0.1.0"
 authors = ["Victor Berger <victor.berger@thalesgroup.com>"]
 
 [dependencies]
-wayland-server = "0.8.4"
+wayland-server = { version = "0.8.4", features = ["dlopen"] }
```

Comment:
Maybe we can accept to rename the feature into something explicit and adopt some convention, I'm thinking something like:

- for the backends: `backend_glutin`, `backend_x11`, `backend_wayland`, `backend_tty`
- for the opengl apis: `renderer_glium`, `renderer_gfx`, (maybe `renderer_vulkano` some time in the future ? ^^)

And in general, a pattern like `<function>_<implementation name>` for every feature of this kind.